Manuals for AKAI MPK Mini 2 and APC Mini (#152)
mixxxdj/manual: source/hardware/controllers/akai_apc_mini.rst

> AKAI APC Mini

Reviewer: I think the manufacturer now uses "Akai" instead of all-uppercase letters in written text.

```suggestion
Akai APC Mini
```

Author: On the website I seen AKAI always written uppercase, it' printed uppercase on the product themselves, but in the FAQ they write Akai... they need to make peace with their own brains.
Anyway, let's write it lowercase :)

===================

> The *AKAI APC Mini* is a controller for a 64 Samplers deck.

Reviewer: ```suggestion
The *Akai APC Mini* is a controller for a 64 Samplers deck.
```

-  `Manufacturer’s product
   page <https://www.akaipro.com/apc-mini>`__

Mapping description
-------------------

> -  The 64 pads are mapped directly to the 64 Samplers available in the extended skins, if the Skin has less Samplers, the non matching buttons will simply not work.

Reviewer: ```suggestion
-  The 64 pads are mapped directly to the 64 Samplers available in the extended skins, if the Skin has less Samplers, the corresponding buttons will simply not work.
```

> -  All buttons lights up in yellow when a sample is configured in the corresponding slot

Reviewer: ```suggestion
-  All buttons light up in yellow when a sample is loaded in the corresponding slot. 
```

> -  When pressed and while playing, they will linght green.

Reviewer: ```suggestion
-  When pressed and while playing, they will light up green.
```

> -  When stopped, they will light in red.

Reviewer: ```suggestion
-  When stopped, they will light up in red.
```

Reviewer: Btw, the LED states would probably make sense to be put into a table for better readability.

Author: Correct, will do

Author: Done!

Akai APC Mini
===================

The *Akai APC Mini* is a controller for a 64 Samplers deck.

> -  `Manufacturer’s product
>    page <https://www.akaipro.com/apc-mini>`__

Reviewer: ```suggestion
-  `Manufacturer’s product page <https://www.akaipro.com/apc-mini>`__
```

Mapping description
-------------------

> 

Reviewer: Here's are some vector images of the two controllers: Could you edit them in Inkscape and add number labels to the same locations as in the Akai user manuals, and then add lists what each numbered control is supposed to do?

Because for me it's still a bit hard to understand how the mapping is working.
[akai-schematic.zip](https://github.com/mixxxdj/manual/files/5384681/akai-schematic.zip)

Author: Sorry for being late, I'll do soon

Reviewer: Ping

Author: Pong!

Changes done, both images are in with numbers, syntax for buttons added... trailing spaces trimmed. Hope that's all :)
